ucore/install-ucore-minimal.sh

```diff
@@ -87,3 +87,5 @@ export IMAGE_NAME=ucore-minimal
 
 # tweak os-release
 sed -i '/^PRETTY_NAME/s/"$/ (uCore minimal)"/' /usr/lib/os-release
+sed -i "s|^VARIANT_ID=.*|VARIANT_ID=$IMAGE_NAME|" /usr/lib/os-release
```

Comment:
This'll make ALL ucore variantes get the VARIANT_ID=ucore-minimal, I don't think we want that.
Also the VARIANT below should be pretty printed, i.e. Variant="uCore minimal"

Reply:
Done

Reply:
Done.